SECURITY-INSIGHTS.yml

```diff
@@ -0,0 +1,55 @@
+header:
+  schema-version: 0.13.4
+  last-updated: '2023-09-05'
+  last-reviewed: '2023-09-05'
+  expiration-date: '2024-09-05T01:00:00.000Z'
+  project-url: https://github.com/WasmEdge/WasmEdge
+  project-release: '0.13.4'
+project-lifecycle:
+  stage: active
```

Comment:
`stage` should be `status`.  https://github.com/ossf/security-insights-spec/blob/main/specification.md#project-lifecycle